lib/elixir/test/elixir/stream_test.exs

```diff
@@ -80,18 +80,23 @@ defmodule StreamTest do
     stream = Stream.chunk_by([1, 2, 2, 3, 4, 4, 6, 7, 7], &(rem(&1, 2) == 1))
 
     assert is_lazy(stream)
-    assert Enum.to_list(stream) ==
```

Comment:
Sorry about that, I just went through each test and add `take` assertion where it missed, to prevent such things in future. Below I added line 85.